crates/router/src/connector/boku/transformers.rs

```diff
@@ -112,47 +268,96 @@ impl<F, T>
     }
 }
 
+fn get_response_status(status: String) -> enums::AttemptStatus {
+    match status.as_str() {
+        "Success" => enums::AttemptStatus::Charged,
+        "Failure" => enums::AttemptStatus::Failure,
+        _ => enums::AttemptStatus::Unresolved,
```

Comment:
Why are we mapping this to `enums::AttemptStatus::Unresolved`? What are all the other possible status?

Reply:
Had a discussion,  changing it to pending👍